.github/workflows/deploy-sage-account-web.yml

```diff
@@ -3,9 +3,11 @@ name: "Deploy SageAccountWeb"
 on:
   push:
     branches:
-      - SageAccountWeb-production
+      - production
+      - staging
       - main
-
+    paths:
+      - "apps/SageAccountWeb/**"
```

Comment:
Changes in upstream dependency `synapse-react-client` won't be picked up by this path